lib/react_client/react_interop.dart

```diff
@@ -39,8 +39,6 @@ abstract class ReactDom {
 /// __DEPRECATED.__
```

Comment:
This notice needs to be removed, as do the deprecations within `react_dom_server.dart`